.travis.yml

```
@@ -1,35 +1,79 @@
+version: ~> 1.0
 language: scala
 
-before_install: curl -Ls https://git.io/jabba | bash && . ~/.jabba/jabba.sh
+before_install: curl -Ls https://raw.githubusercontent.com/shyiko/jabba/0.11.2/install.sh | bash && . ~/.jabba/jabba.sh
 install: jabba install "adopt@~1.$TRAVIS_JDK.0-0" && jabba use "$_" && java -Xmx32m -version
 
 git:
   depth: false # Avoid sbt-dynver not seeing the tag
 
 env:
   matrix:
-    - TRAVIS_JDK=8
     - TRAVIS_JDK=11
 
+  global:
+    # encrypt with: travis encrypt --pro BINTRAY_USER=...
+    - secure: "EFLixFgHISkKkjxk6FV1WMSUcYSZObsPL/055PALBesySLRxrosy7dU2lbESSRSJOYrLV79MJkT1LdiOdDMUKACu4STIDiC46YMkffa+VdWsUIhB5uktSa5/RfLTj2LGIHsygDPD2YcoIqz2WJus34wK88fdRYga3fnH23ilnSg="
+    # encrypt with: travis encrypt --pro BINTRAY_PASS=...
+    - secure: "S86NIK95mKMSJd5XZ1PjWNC0Dg8Pmr6oSdTJB5AKSTkeWRlkuE4ldkz841gYk6QObCcVT/mAIV5i8QW4KK+Md/4fq+eExEoe1VBAMSglBQyzeLFaKbPL7aKsk7Ma+Qeis5nFU6dje/ANuK4mAWxAmORq8r5+h5oT5uIb+UrOZn0="
+
 scala:
-  - 2.12.10
-  - 2.13.1
+  - 2.13.2
+
+script: sbt "++$TRAVIS_SCALA_VERSION" test
```

since we removed 2.12 from the scala matrix, shouldn't we change this to +test?

```suggestion
script: sbt +test
```

Or are we confident testing only on 2.13 will be sufficient?

---

or is this script overwritten by the jobs?

---

I expected the one the job sets explicitly to apply, but it is actually overwritten by Travis:
https://travis-ci.com/github/playframework/cachecontrol/jobs/346631266#L197
(Now I wonder if we got this wrong on other repos...)